content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -76,8 +83,12 @@ const express = require("express");
 const PORT = parseInt(process.env.PORT || "8080");
 const app = express();
 
-app.get("/", (req, res) => {
-  res.send("Hello World");
+function getRandomNumber(int: min, int: max) {
```

Comment:
I think this would be the only example using Flow right now, right? :thinking:

```suggestion
function getRandomNumber(min, max) {
```

Reply:
oh... that shouldn't be there anymore, thanks!